### PR TITLE
Updating to clarify Gitlens+ accounts

### PR DESCRIPTION
**Pull request by Axxzilla**

### CODE REVIEW

File: gitlens/gitlens-account.md [hunk 1]
```diff
@@ -9,6 +9,14 @@ taxonomy:
 
 Whether a solo user on a personal license, or an owner managing multiple teams across organizations, learn how to manage your GitKraken GitLens account(s), subscription(s), and organization(s).
 
+The first question to answer: do I need an account to use GitLens or GitLens+? 
+
++ GitLens is COMPLETELY FREE, no account is required, and can be used on any local repository.
++ GitLens+ features on local & public repositories are also COMPLETELY FREE.
++ GitLens+ features on private repositories require a Pro plan (or greater), view pricing [here](https://www.gitkraken.com/gitlens/pricing).
+
+If you are working on GitLens+ on private repositories, you will want to create an acount and purchase a license. See below for more information.
```
Comment (jkelroy) [inline gitlens/gitlens-account.md:18]: This is oddly worded. Maybe something like this?
> If you are working ~on~ with GitLens+ features on private repositories, you will ~want~ need to create...

File: gitlens/gitlens-account.md [hunk 2]
```diff
@@ -9,6 +9,14 @@ taxonomy:
 
 Whether a solo user on a personal license, or an owner managing multiple teams across organizations, learn how to manage your GitKraken GitLens account(s), subscription(s), and organization(s).
 
+The first question to answer: do I need an account to use GitLens or GitLens+? 
+
++ GitLens is COMPLETELY FREE, no account is required, and can be used on any local repository.
```
Comment (jkelroy) [inline gitlens/gitlens-account.md:14]: Although I understand this is for emphasis, I think the use of capital words in these docs (and the other page) are a bit out of place. If we did want to emphasize these words, maybe bold is a better option?

**Review by jkelroy — APPROVED** [review]

File: gitlens/gitlens-account.md (hunk 2 above)
Comment (Axxzilla) [reply gitlens/gitlens-account.md:14]: Changed to be **bold** instead

File: gitlens/gitlens-account.md (hunk 1 above)
Comment (Axxzilla) [reply gitlens/gitlens-account.md:18]: yeah, sounds better

File: gitlens/gitlens-account.md [hunk 3]
```diff
@@ -9,6 +9,14 @@ taxonomy:
...
++ GitLens+ features on local & public repositories are also COMPLETELY FREE.
++ GitLens+ features on private repositories require a Pro plan (or greater), view pricing [here](https://www.gitkraken.com/gitlens/pricing).
+
+If you are working with GitLens+ features on private repositories, you will need to create an acount and purchase a license. See below for more information.
```
Comment (amalyshkin) [inline gitlens/gitlens-account.md:18]: "... create an ac**C**ount ..."

**Pull request merged**